### DISCUSSION TITLE
Problème sur MySQL

### DISCUSSION THREAD

**Discussion by RaphaelCohen1** [community/community#116529] [Programming Help]
### Select Topic Area

Question

### Body

SELECT * FROM `Age`
WHERE >= 3 AND 40 AND code_postal = '94';

Ma console indique "#1064 - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '>= 3 AND 40  AND code_postal = '94' LIMIT 0, 25' at line 2" qu'elle est mon erreur?

**Comment by orsiczako** [comment]
I think the problem is that you tried to write a condition where you say something is equal or bigger than 3, however you didn't mention which column has to be euqal or bigger in the AGE table. It should look like this:
SELECT * FROM Age 
WHERE column_name>=3 
AND column_name=40 
AND code_postal='94';
I translated it for you in case you wouldn't understand it:
Je pense que le problème est que vous avez essayé d'écrire une condition dans laquelle vous dites que quelque chose est égal ou supérieur à 3, mais vous n'avez pas mentionné quelle colonne doit être euqal ou plus grande dans la table AGE. Ça devrait ressembler à ça:
SELECT * FROM Age 
WHERE column_name>=3 
AND column_name=40 
AND code_postal='94';
I hope this helped and this is the outcome you wanted.
